.circleci/config.yml

```diff
@@ -38,7 +39,7 @@ jobs:
       - run:
           name: Generate Client Hash For This Build
           command: |
-            find ~/project/app/client/ ~/project/app/package.json -type f -print0 | sort -z | xargs -0 sha1sum | sha1sum  > ~/project/ThisClientHash.txt
+            find ~/project/app/client/ ~/project/app/package.json -type f -print0 | sort -z | xargs -0 sha1sum | sha1sum | head -n1 | awk '{print $1;}'  > ~/project/ThisClientHash.txt
```

Comment:
Hash used to come out as `0194467ff7a39fdb0ff5e4365065b7989e0c27ce -`. This trims the dash so now it's just `0194467ff7a39fdb0ff5e4365065b7989e0c27ce`. This wasn't a problem, but also not technically correct